### DISCUSSION TITLE
EDDI + VA Reporting first footfall?

### DISCUSSION THREAD

**Discussion by Tychious** [EDCD/EDDI#2529] [EDDI VoiceAttack Q&A]
Is there any variable to check if a planet has had a footfall on it already? 

I have seen there was the same question asked but it was a year ago. Not sure if anyone has stumbled upon this.

**Comment by Tkael** [comment]
I'm afraid the answer remains the same. As far as I'm aware there is no information available to indicate a first footfall event.
